src/components/Simulering/SimuleringPensjonsavtalerAlert/SimuleringPensjonsavtalerAlert.tsx

```diff
@@ -63,20 +63,22 @@ export const SimuleringPensjonsavtalerAlert: React.FC<Props> = ({
 
     if (isEndring) {
       const text = 'beregning.pensjonsavtaler.alert.endring'
```

Comment:
istedenfor å gjenta `variant: 'inline-info'`, linje 68 og linje 71, ville jeg skrive det i en konstant, slik det er gjort for `tekst` linje 67.

Det gjelder for hver kall til logger() i denne filen.